proto/pocket/migration/query.proto

```diff
@@ -58,3 +72,20 @@ message QueryAllMorseClaimableAccountResponse {
            cosmos.base.query.v1beta1.PageResponse pagination            = 2;
 }
 
+// QueryMorseClaimableAccountRequest is used to query for MorseClaimableAccounts by Morse address.
+// If the shannon_address (or claimed_at_height) for any given MorseClaimableAccount
+// is empty, it has not been claimed.
+// See the comments for MorseClaimableAccount for more context.
+message QueryMorseClaimableAccountsByShannonAddressRequest {
```

Comment:
Don't do this, instead add a `oneof filter` to `QueryMorseClaimableAccounts` instead.